mk/platform.mk

```diff
@@ -1,6 +1,10 @@
 ifeq ($(OSTYPE),auto)
 OSTYPE=$(shell uname | tr 'A-Z' 'a-z')
 endif
+ifneq (,$(findstring darwin,${OSTYPE}))
```

Comment:
use ifneq($(OSTYPE),darwin)

.. but why? i think this will break on BSDs. and this sounds more like a linuxism

Reply:
Hmmm in fact this change was accidental....